packages/ui/tailwind-utils-config/components/textarea.ts

```diff
@@ -0,0 +1,104 @@
+import { CSSRuleObject } from 'tailwindcss/types/config'
+
+// themes can be extended in the future
+const themes = ['danger'] as const
+
+function createInputThemeStyles() {
+  const styles: CSSRuleObject = {}
+
+  themes.forEach(theme => {
+    styles[`&:where(.cn-textarea-${theme}):not(textarea[disabled])`] = {
+      borderColor: `var(--cn-border-${theme})`,
+      boxShadow: `var(--cn-ring-${theme})`,
+
+      '&:where(:hover)': {
+        borderColor: `var(--cn-border-${theme})`,
+        boxShadow: `var(--cn-ring-${theme}-hover)`
+      },
+
+      '&:where(:focus)': {
+        borderColor: `var(--cn-border-${theme})`,
+        boxShadow: `var(--cn-ring-${theme})`
+      }
+    }
+  })
+
+  return styles
+}
+
+export default {
+  '.cn-textarea': {
+    borderRadius: 'var(--cn-input-radius)',
+    padding:
+      'var(--cn-input-default-py) var(--cn-input-default-pr) var(--cn-input-default-py) var(--cn-input-default-pl)',
+    minHeight: '34px', // about 1 line of text
+    borderWidth: 'var(--cn-input-border)',
+    borderStyle: 'solid',
+    borderColor: 'var(--cn-border-2)',
+    backgroundColor: 'var(--cn-bg-2)',
+    '@apply font-body-normal': '',
+    color: 'var(--cn-text-1)',
+    whiteSpace: 'pre-wrap',
+    wordBreak: 'break-word',
+    resize: 'none',
```

Comment:
why disallow resize forcefully? i think this goes against the principles of accessibility

Reply:
We have a prop called `resizeable` in the [component API](https://gist.github.com/knagurski/7ce33d8c24f925e1d2f06c1729e1f18b), so this prop enables the resize if passed